diff --git a/sshuttle/methods/pf.py b/sshuttle/methods/pf.py
index cab5b1c..e89f8a6 100644
--- a/sshuttle/methods/pf.py
+++ b/sshuttle/methods/pf.py
@@ -214,8 +214,8 @@ def add_rules(self, anchor, includes, port, dnsport, nslist, family):

         tables = []
         translating_rules = [
-            b'rdr pass on lo0 %s proto tcp from ! %s to %s '
-            b'-> %s port %r' % (inet_version, lo_addr, subnet, lo_addr, port)
+            b'rdr pass on lo0 %s proto tcp to %s '
+            b'-> %s port %r' % (inet_version, subnet, lo_addr, port)
             for exclude, subnet in includes if not exclude
         ]
         filtering_rules = [
